packages/blockchain-wallet-v4-frontend/src/scenes/Home/BalancesChart/selectors.js

```diff
@@ -21,12 +21,19 @@ export const getBchBalance = state => createDeepEqualSelector(
   ],
   (context) => {
     const getBalance = address => selectors.core.data.bch.getFinalBalance(address, state)
-    const balancesR = traverse(Remote.of, getBalance, context)
-    return balancesR.map(xs => reduce(add, 0, xs))
+    const balancesR = context.map(x => getBalance(x).getOrElse(0))
+    return Remote.of(reduce(add, 0, balancesR))
   }
 )(state)
 
-export const getEthBalance = selectors.core.data.ethereum.getBalance
+export const getEthBalance = state => createDeepEqualSelector(
+  [
+    selectors.core.data.ethereum.getBalance
+  ],
+  (balance) => {
+    return Remote.of(balance.getOrElse(0))
+  }
+)(state)
 
```

Comment:
We've gone back and forth on this one a few times. We need to `getOrElse(0)` for things like total balances.

Reply:
Ok my bad !

Reply:
No worries :)